metaflow/runtime.py

```diff
@@ -770,6 +785,7 @@ def __init__(self, task, max_logs_size):
 
         self._encoding = sys.stdout.encoding or 'UTF-8'
         self.killed = False
+        self.cleaned = False
```

Comment:
Could you add a short comment on what self.killed=True represents vs self.cleaned=True?
It also seems if self.killed = True, then self.cleaned must be True?

Defining the state transition perhaps in a short comment helps understand why you are tracking two states.